app/assets/stylesheets/common/_markus.scss

```diff
@@ -184,10 +175,10 @@ input[type='submit'],
 input[type='reset'],
 button,
 .button {
-  @include background-gradient($white, $light-grey);
-  @include border-radius($radius);
-  @include box-shadow(inset 0 -1px 0 $grey);
-  @include transition(all, $time-quick);
+  background: linear-gradient(top, $white, $light-grey);
+  background-color: $white;
+  border-radius: $radius;

```

Comment:
Properties should be ordered background, background-color, border, border-radius, box-shadow, color, cursor, display, font, min-width, outline, padding, text-align, text-shadow, transition